[2025-03-14T18:24:38.835+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T18:24:38.841+0000] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: open_weather_etl.load manual__2025-03-14T18:24:37.113603+00:00 [queued]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0, success_setup=0, skipped_setup=0), upstream_task_ids={'transform'}
[2025-03-14T18:24:38.841+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
